src/go/docker-credential-none/cmd/root.go

```diff
@@ -51,13 +52,22 @@ using the same format that docker uses when no credsStore field is specified in
 This helper is intended for testing purposes, but will be used on Linux systems
 unless 'pass' and/or 'secretservice' is available.`,
 		configFileName),
+	DisableSuggestions: true,
+	SilenceUsage:       true,
```

Comment:
That seems wrong — usage should not be suppressed if there's a parse error (when being used interactively)?

I can see why that might be not great when using over the API, though…